playground/demo.js

```diff
@@ -1,13 +1,15 @@
 function demo({ Client, Query, Zone }) {
   const client = new Client({ zone: Zone.EU1 });
-  const query = new Query({hashid: 'c0604b71c273c1fb3ef13eb2adfa4452'});
+  const request = new Query({hashid: 'c0604b71c273c1fb3ef13eb2adfa4452'});
 
-  query.searchText('silla');
-  query.addFilter('brand', 'JANE');
-  query.addFilter('brand', 'RECARO');
-  query.queryCounter = 2;
+  request.query = 'silla';
```

Comment:
I like more request.searchText because "query" is a whidest concept.

Reply:
I will use `request.text` then.